avocado/utils/network/interfaces.py

```diff
@@ -264,17 +264,19 @@ def save(self, ipaddr, netmask):
             msg = 'Distro not supported by API. Could not save ipaddr.'
             raise NWException(msg)
 
-        self._write_to_file("{}/{}".format(path, filename),
-                            {'TYPE': self.if_type,
-                             'BOOTPROTO': 'none',
-                             'NAME': self.name,
-                             'DEVICE': self.name,
-                             'ONBOOT': 'yes',
-                             'IPADDR': ipaddr,
-                             'NETMASK': netmask,
-                             'IPV6INIT': 'yes',
-                             'IPV6_AUTOCONF': 'yes',
-                             'IPV6_DEFROUTE': 'yes'})
+        ifcfg_dict = {'TYPE': self.if_type,
+                      'BOOTPROTO': 'none',
+                      'NAME': self.name,
+                      'DEVICE': self.name,
+                      'ONBOOT': 'yes',
+                      'IPADDR': ipaddr,
+                      'NETMASK': netmask,
+                      'IPV6INIT': 'yes',
+                      'IPV6_AUTOCONF': 'yes',
+                      'IPV6_DEFROUTE': 'yes'}
+        if current_distro.name == 'SuSE':
+            ifcfg_dict.update({'BOOTPROTO': 'none'})
+            self._write_to_file("{}/{}".format(path, filename), ifcfg_dict)
```

Comment:
Did you tested this on a non-SuSE distro? Because your change just removed support to other distros. Maybe an indentation problem on line 279 here.

Reply:
Also, please see the Travis errors.